### PR TITLE
Sarbast's Node HW week1

### PR DESCRIPTION
**Pull request by SarKurd**

### CODE REVIEW

File: week1/homework/src/server.js [hunk 1]
```diff
@@ -10,6 +10,32 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('Content-Type', 'application/json');
+        response.write(`{"state": ${state}}`);
```
Comment (alxbrg) [inline week1/homework/src/server.js]: I can tell you understand what a JSON string is, that's the most important, but we have a nice little method for making a JSON string out of a JS object: [`JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).

File: week1/homework/src/server.js [hunk 2]
```diff
@@ -10,6 +10,32 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('Content-Type', 'application/json');
```
[the remark this answers is not in the document]
Comment (SarKurd) [reply week1/homework/src/server.js]: I should have done that but what if we have more than one different  `Content-Type`?

will update the code

File: week1/homework/src/server.js (hunk 1 above)
Comment (SarKurd) [reply week1/homework/src/server.js]: I'm well aware of that but the code was short so i though it's not necessary, if you insist i'll update the code.

File: week1/homework/src/server.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (GeorgeSapkin) [reply week1/homework/src/server.js]: Then, of course, you have to do it per route. But this is not the case with this assignment.

[the remark this answers is not in the document]
Comment (SarKurd) [reply week1/homework/src/server.js]: I'll update the code now.

Thank you all

**Pull request closed without merging**